src/Microsoft.Diagnostics.Runtime/src/Linux/Structs/ElfFileTableHeader64.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Diagnostics.Runtime.Linux
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public struct ElfFileTableHeader64
```

Comment:
Should this be public?  I think `internal` is more appropriate unless you need it for something?

Reply:
Yes all these ELF structures should be internal.  I make the changes.